src/client/ldBrowser.tsx

```diff
@@ -26,6 +26,7 @@ class LDBrowser extends Component<LDBrowserProps, HocState> {
     this.state = {
       flags: ssrFlags,
       ldClient,
+      user: ldUser,
```

Comment:
The user here needs to be updated on `identify()` calls, otherwise this will always be the first user. However I think this is a separate issue from this PR and is something I'll internally discuss and fix.